packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -1172,4 +1173,25 @@ export class RpcClient implements RpcClientInterface {
       method: 'GET',
     });
   }
+
+  /**
+   *
+   * @param params contains the kind (PVM) and kernel to generate the origination proof from
```

Comment:
I think it's a bit more clear to say 'PVM kind' rather than 'kind (PVM)', we could also ditch the 'from' at the end